src/AzSK.ADO/Framework/Abstracts/ADOSVTBase.ps1

```diff
@@ -2,6 +2,7 @@ class ADOSVTBase: SVTBase {
 
 	hidden [ControlStateExtension] $ControlStateExt;
 	hidden [AzSKSettings] $AzSKSettings;
+	hidden [bool] $isResourceActive = $true;
```

Comment:
This should be reset for each resource individually.

Reply:
add comment

Reply:
Value for this variable will be overridden for each and every resource